spring-cloud-dataflow-server-local/src/test/java/org/springframework/cloud/dataflow/server/local/security/SecurityTestUtils.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.dataflow.server.local.security;
+
+import org.springframework.security.crypto.codec.Base64;
+
+/**
+ * @author Marius Bogoevici
+ * @author Gunnar Hillert
+ */
+public class SecurityTestUtils {
+	public static String basicAuthorizationHeader(String username, String password) {
+		return "Basic " + new String(Base64.encode((username + ":" + password).getBytes()));
```

Comment:
Should we change the `getBytes()` call?
http://stackoverflow.com/a/7243567/1662330